packages/now-python/src/index.ts

```diff
@@ -71,14 +71,14 @@ async function pipInstallUser(pipPath: string, ...args: string[]) {
 }
 
 async function pipenvInstall(pyUserBase: string, srcDir: string) {
-  console.log('running "pipenv_to_requirements -f');
+  console.log('running pipfile2req');
   try {
-    await execa(join(pyUserBase, 'bin', 'pipenv_to_requirements'), ['-f'], {
+    const out = await execa.stdout(join(pyUserBase, 'bin', 'pipfile2req'), [], {
```

Comment:
```suggestion
    const out = await execa.stdout(join(pyUserBase, 'bin', 'pipfile2req'), ['--no-warn-script-location'], {
```

I think this should work